client/.gitignore

```diff
@@ -22,3 +22,12 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+# Yarn directories
+node_modules/*
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/versions
```

Comment:
Followed docs from https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored